ERCS/erc-7660.md

```diff
@@ -0,0 +1,426 @@
+---
+eip: 7660
+title: Release tokens on predefined periods
+description: Controlled by smart contracts, tokens can be linearly released based on the predefined period.
```

Comment:
Your description doesn't have any information not common to all ERCs and not already included in your title: just "linearly released".

Reply:
have changed

Reply:
ok ,we changed